federation/pkg/kubefed/util/util.go

```diff
@@ -254,9 +254,9 @@ func GetVersionedClientForRBACOrFail(hostFactory cmdutil.Factory) (client.Interf
 				}
 				return hostFactory.ClientSetForVersion(&gv)
 			}
-			for i := 0; i < len(g.Versions); i++ {
-				if g.Versions[i].GroupVersion != "" {
-					gv, err := schema.ParseGroupVersion(g.Versions[i].GroupVersion)
+			for _, version := range g.Versions {
+				if version.GroupVersion != "" {
+					gv, err := schema.ParseGroupVersion(version.GroupVersion)
```

Comment:
Sorry to say this ... but I don't think this make too much difference. Do you have reason to do this, for example, code style best practice?